packages/rest/src/rest.server.ts

```diff
@@ -190,6 +191,15 @@ export class RestServer extends Context implements Server, HttpServerLike {
     this._setupRequestHandler();
 
     this.bind(RestBindings.HANDLER).toDynamicValue(() => this.httpHandler);
+
+    // LB4's static assets serving router
+    const staticAssetsRouter = new Route('get', '*', {responses: {}}, () => {
+      console.log('>> staticAssetsRouter HANDLER');
+      // @bajtos How do we access req and res here?
```

Comment:
You cannot access the request/response from a handler route (a `Route` instance). In my comment https://github.com/strongloop/loopback-next/issues/1784#issuecomment-428916153, I was showing how to create a new kind of `RouteEntry` (a new class implementing `RouteEntry` interface). RouteEntry's method `invokeHandler` does have access to the request/response objects.

```ts
  // somewhere in the code setting up RoutingTable
  // the anonymous RouteEntry object should be refactored into a class
  this.routingTable.catchAllRoute = <RouteEntry>{
    verb: 'get',
    path: '*',
    spec: { /* TODO: this route should be excluded from the spec */ },
    updateBindings(requestContext: Context) {
      // no-op
    },
    
    invokeHandler(
      {request, response}: Context,
      args: OperationArgs,
    ): Promise<OperationRetval> {
      return this._handleStaticAssetRequest(request, response);
    }

    describe(): string {
      return "final route to handle static assets";
    }
  };
```